interactives/models/hawk-rabbit-combo.coffee

```diff
@@ -608,6 +608,15 @@ window.model =
     # Once there are 50 rabbits, they will stop reproducing entirely
     @setProperty(allRabbits, "mating chance", -.005 * @numRabbits + .25)
 
+    if @addedRabbits and @addedHawks
+        that = @
+        allRabbits.forEach((rabbit) -> 
+          if (rabbit.get('color') != that.envColors[location.index])
+            # When rats are getting preyed on, there should be less of them
+            # Reduce their carrying capacity here to accomplish that
+            rabbit.set('mating chance', -.005 * that.numRabbits + .22)
```

Comment:
I realized that the population does already dip much more frequently with hawks in place and a visible population...decreasing the mating chance even a little like this seems to make the effect substantially more noticeable.

Reply:
I think it looks ok. It's basically an artifact of the hawks not being everywhere at once, but a clustering effect that happens, as the hawks clear out one area and the population explodes on the other side of the field, and then repeat. Since it's very visible what's happening on the screen, and since the average population *does* decrease, I think it's fine.